test/exe/version_out_test.sh

```diff
@@ -6,23 +6,29 @@ set -e -o pipefail
 export LC_ALL=C
 
 ENVOY_BIN="${TEST_SRCDIR}/envoy/source/exe/envoy-static"
+REGEX=".*version: ([0-9a-f]{40})\/([0-9]+\.[0-9]+\.[0-9]+)-([0-9]+)-g([0-9a-f]{10})(-dirty)?\/(Clean|Modified)\/(RELEASE|DEBUG)\/([a-zA-Z-]+)$"
 
-COMMIT=$(${ENVOY_BIN} --version | \
-  sed -n -E 's/.*version: ([0-9a-f]{40})\/([0-9]+\.[0-9]+\.[0-9]+)(-[a-zA-Z0-9\-_]+)?\/(Clean|Modified)\/(RELEASE|DEBUG)\/([a-zA-Z-]+)$/\1/p')
+COMMIT=$(${ENVOY_BIN} --version | sed -n -E "s/${REGEX}/\1/p")
+EXPECTED_COMMIT=$(cat "${TEST_SRCDIR}/envoy/bazel/raw_build_id.ldscript")
 
-EXPECTED=$(cat "${TEST_SRCDIR}/envoy/bazel/raw_build_id.ldscript")
-
-if [[ "${COMMIT}" != "${EXPECTED}" ]]; then
-  echo "Commit mismatch, got: ${COMMIT}, expected: ${EXPECTED}".
+if [[ "${COMMIT}" != "${EXPECTED_COMMIT}" ]]; then
+  echo "Commit mismatch, got: ${COMMIT}, expected: ${EXPECTED_COMMIT}".
   exit 1
 fi
 
-VERSION=$(${ENVOY_BIN} --version | \
-  sed -n -E 's/.*version: ([0-9a-f]{40})\/([0-9]+\.[0-9]+\.[0-9]+)(-[a-zA-Z0-9\-_]+)?\/(Clean|Modified)\/(RELEASE|DEBUG)\/([a-zA-Z-]+)$/\2\3/p')
+VERSION=$(${ENVOY_BIN} --version | sed -n -E "s/${REGEX}/\2/p")
+EXPECTED_VERSION=$(cat "${TEST_SRCDIR}/envoy/VERSION")
+
+if [[ "${VERSION}" != "${EXPECTED_VERSION}" ]]; then
+  echo "Version mismatch, got: ${VERSION}, expected: ${EXPECTED_VERSION}".
+  exit 1
+fi
 
-EXPECTED=$(cat "${TEST_SRCDIR}/envoy/VERSION")
+ABBREV_COMMIT=$(${ENVOY_BIN} --version | sed -n -E "s/${REGEX}/\4/p")
```

Comment:
nit: I think it's called `label` in the code: https://github.com/10gen/envoy-serverless/blob/51a76ca5b916d9e992ee5a4742f653a9ff88d926/source/common/version/version.cc#L73